### PR TITLE
Results reviewer request part1

### PR DESCRIPTION
**Pull request by danich1**
This is the first part of changes for the result section

Main changes:
1. Incorporated supplemental analysis with single and special characters removed
2. Increased the label size for both figures
3. Move entire "Document embeddings derived from bioRxiv reveal fields and subfields" section into the supplement
4. Added two tables of cosine similarity scores along with the words as requested by reviewer

### CODE REVIEW

File: content/04.results.md [hunk 1]
```diff
@@ -41,75 +41,23 @@ We plotted the 95% confidence interval for each reported token.
...
 These findings support our notion that bioRxiv is linguistically similar to the PMCOA repository.
 
-Terms like "neurons", "genome", and "genetic", which are common in genomics and neuroscience, were more common in bioRxiv than PMCOA while others associated with clinical research, such as "clinical" "patients" and "treatment" were more common in PMCOA (Figure {@fig:corpora_comparison_panels}B and {@fig:corpora_comparison_panels}C).
-When controlling for the differences in the body of documents to identify textual changes associated with the publication process, we found that tokens such as "et" "al" were enriched for bioRxiv while "$\pm$", "–" were enriched for PMCOA (Figure {@fig:corpora_comparison_panels}D and {@fig:corpora_comparison_panels}E).
+Terms like "neurons", "genome", and "genetic", which are common in genomics and neuroscience, were more common in bioRxiv than PMCOA while others associated with clinical research, such as "clinical" "patients" and "treatment" were more common in PMCOA (Figure {@fig:corpora_comparison_panels}B, {@fig:corpora_comparison_panels}C and Supplementary Figure {@fig:corpora_comparison_panels_bio_v_pmid}).
```
Comment (cgreene) [inline content/04.results.md:52]:
```suggestion
The terms "neurons", "genome", and "genetic", which are common in genomics and neuroscience, were more common in bioRxiv than PMCOA while others associated with clinical research, such as "clinical" "patients" and "treatment" were more common in PMCOA (Figure {@fig:corpora_comparison_panels}B, {@fig:corpora_comparison_panels}C and Supplementary Figure {@fig:corpora_comparison_panels_bio_v_pmid}).
```

File: content/04.results.md [hunk 2]
```diff
@@ -41,75 +41,23 @@ We plotted the 95% confidence interval for each reported token.
...
+When controlling for the differences in the body of documents to identify textual changes associated with the publication process, we found that tokens such as "et" "al" were enriched for bioRxiv while "$\pm$", "–" were enriched for PMCOA (Figure {@fig:corpora_comparison_panels}D, {@fig:corpora_comparison_panels}E). 
+When removing special and single-character tokens, more figure-related terms appeared enriched for published articles, and more research-related articles appeared enriched for bioRxiv (Supplementary Figure {@fig:corpora_comparison_panels_pre_v_pub}).
```
Comment (cgreene) [inline content/04.results.md:54]: What is a "research-related article"? I am missing something. Do you mean terms associated with research vs terms associated with presentation (i.e., fig vs figure)? I am a bit confused here.

File: content/90.back-matter.md [hunk 3]
```diff
@@ -3,56 +3,133 @@
...
+**D.** Examining PC1 values for each article by category created a continuum from informatics-related fields on the top through cell biology on the bottom.
+Specific article categories (neuroscience, genetics) were spread throughout PC1 values.
+**E.** Examining PC2 values for each article by category revealed fields like genomics, bioinformatics, and genetics on the top and neuroscience and behavior on the bottom.
+](https://raw.githubusercontent.com/danich1/annorxiver/5a593f0250886192650dd329e6968069b13e4fcb/figure_generation/output/figure_two_panels.png){#fig:topic_analysis_panels width="100%"}
+
+Document embeddings provide a means to categorize the language of documents in a way that takes into account the similarities between terms [@arxiv:1405.4053; @arxiv:1707.02377; @arxiv:2001.05727].
+We found that the first two PCs separated articles from different author-selected categories (Figure {@fig:topic_analysis_panels}A).
+Certain neuroscience papers appeared to be more associated with the cellular biology direction of PC1, while others seemed to be more associated with the informatics-related direction Figure {@fig:topic_analysis_panels}A).
```
Comment (cgreene) [inline content/90.back-matter.md:28]: If this is in the supplement, I think these also have to be supplementary figures. Can you remind me - we moved this to address a reviewer concern that the paper was too long and complicated?

File: content/90.back-matter.md [hunk 4]
```diff
@@ -3,56 +3,133 @@
...
+This suggests that the concepts captured by PCs were not exclusively related to their field.
+
+Visualizing token-PC similarity revealed tokens associated with certain research approaches (Figures {@fig:topic_analysis_panels}B and {@fig:topic_analysis_panels}C).
```
Comment (cgreene) [inline content/90.back-matter.md:31]: All of these need to be sup fig names / panels too.

File: content/90.back-matter.md [hunk 5]
```diff
@@ -3,56 +3,133 @@
...
+
+Examining the top five highest-scoring and bottom five lowest-scoring systems biology preprints along PC1 reinforces its dichotomous theme (Supplementary Table {@tbl:five_pc1_table}).
+Preprints with the highest values [@doi:10.1101/197400;@doi:10.1101/825943;@doi:10.1101/044818;@doi:10.1101/769299;@doi:10.1101/107250] included software packages, machine learning analyses, and other computational biology manuscripts, while preprints with the lowest values [@doi:10.1101/455048;@doi:10.1101/371922;@doi:10.1101/733162;@doi:10.1101/745943;@doi:10.1101/754572] were focused on cellular signaling and protein activity.
+We provide the rest of our 50 generated PCs in our online repository (see Software and Data Availability).
+
+| Title [citation]     | PC1  | License | Figure Thumbnail |
+|--------------------------------|-------------------|------------|----------|
+| Conditional Robust Calibration (CRC): a new computational Bayesian methodology for model parameters estimation and identifiability analysis [@doi:10.1101/197400] | 4.522818390064091 | None         | ![](images/paper-thumbnails/pc1/197400_thumbnail.png) |
```
Comment (cgreene) [inline content/90.back-matter.md:47]: I really thought this analysis was cool 😭 . Even if this gets sent to the supplement, we need to refer readers to the supplement somewhere. What do you think about adding a few sentences to the end of the first part of the results section that summarize this section and point folks here if they are interested?

**Review by cgreene — COMMENTED** [review]
If we do move these things, we will have to let readers know they exist here in the supplement and you'll have some figure renumbering to do!

File: content/90.back-matter.md (hunk 3 above)
Comment (danich1) [reply content/90.back-matter.md:28]: >I would cut this entire subsection from the manuscript to make it shorter (or relegate it to a supplementary section).
>
>Don't we already know that if one uses full texts we can determine the subfield of the paper? It's not that interesting in my opinion and not relevant to the main hypotheses of the paper -- comparing between preprints and the journal published version.

The reviewer found that whole section to be not interesting, so they suggested we remove this section or push it to the supplement. I think removing is extreme and supplement would suffice if it has to be changed.

File: content/04.results.md (hunk 2 above)
Comment (danich1) [reply content/04.results.md:54]: >What is a "research-related article"? I am missing something. Do you mean terms associated with research vs terms associated with presentation (i.e., fig vs figure)? I am a bit confused here.

I meant to say terms related to research rather journal presentation. Will update this sentence to be more clear.

File: content/90.back-matter.md (hunk 3 above)
Comment (cgreene) [reply content/90.back-matter.md]: Ok - then I think you definitely need to have a way for folks to find this, so it needs some brief mention of the results and the existence of the analysis in the supplement with a reference to (Supplementary Results).

Comment (cgreene) [reply content/90.back-matter.md]: And all of these figures then need to be referred to / numbered as supplementary figs.

Comment (cgreene) [reply content/90.back-matter.md]: Ahh - I see the new sentence now!

File: content/04.results.md [hunk 6]
```diff
@@ -41,75 +41,27 @@ We plotted the 95% confidence interval for each reported token.
...
-| Spatiotemporal proteomics uncovers cathepsin-dependent host cell death during bacterial infection [@doi:10.1101/455048]                                            | -3.728443135960558  | CC-BY-ND | ![](images/paper-thumbnails/pc1/455048_thumbnail.png)            | 
-| NADPH consumption by L-cystine reduction creates a metabolic vulnerability upon glucose deprivation [@doi:10.1101/733162]                                          | -3.7363965062637288 | None     | ![](images/paper-thumbnails/pc1/733162_thumbnail.png)            | 
-| AKT but not MYC promotes reactive oxygen species-mediated cell death in oxidative culture [@doi:10.1101/754572]                                                    | -3.8769231933681176 | None     | ![](images/paper-thumbnails/pc1/754572_thumbnail.png)            | 
-
-Table: PC1 divided the author-selected category of systems biology preprints along an axis from computational to molecular approaches. {#tbl:five_pc1_table}
+Following our global analysis, we performed principal component analysis on document emebddings generated by the bioRxiv corpus to inspect latent concepts captured within them.
```
Comment (cgreene) [inline content/04.results.md:62]:
```suggestion
Following our analysis of tokens, we examined the principal components of document embeddings derived from bioRxiv.
```

File: content/04.results.md [hunk 7]
```diff
@@ -41,75 +41,27 @@ We plotted the 95% confidence interval for each reported token.
...
+Our analysis revealed that the strongest prinicpal component (PC) separated cell biology and informatics-related fields, while the second strongest component separated bioinformatics concepts from neuroscience concepts.
```
Comment (cgreene) [inline content/04.results.md:63]:
```suggestion
We found that the top principal components separated methodological approaches and research fields.
Preprints from certain topic areas that spanned approaches from informatics-related to cell biology could be distinguished using these principal components (see Supplementary Results).
```

File: content/04.results.md [hunk 8]
```diff
@@ -41,75 +41,27 @@ We plotted the 95% confidence interval for each reported token.
...
+We explore these results further within our supplemental section (see 'Document embeddings derived from bioRxiv reveal fields and subfields' for more information).
```
Comment (cgreene) [inline content/04.results.md:64]:
```suggestion
```

**Review by cgreene — COMMENTED** [review]
Ok - I think now you just need to renumber the supplementary bits unless I didn't see that you had done that.

**Review by cgreene — APPROVED** [review]
Assuming supplement is fully renumbered 👍

**Pull request merged**